cmd/docsync/main.go

```diff
@@ -0,0 +1,169 @@
+package main
```

Comment:
There's a whole style of documenting files that implement commands like this; see `docgenerate` for one such example: https://github.com/chain/chain/blob/main/cmd/docgenerate/main.go